.github/workflows/update-xla-commit-hash.yml

```diff
@@ -0,0 +1,65 @@
+name: Update xla commit hash
+
+on:
+  schedule:
+    # Every day at 12:37am
+    # Choose a random time near midnight because it may be delayed if there are high loads
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron: 37 0 * * *
+  workflow_dispatch:
+
+env:
+  NEW_BRANCH_NAME: update-xla-commit-hash/${{ github.run_id }}-${{ github.run_number }}-${{ github.run_attempt }}
+
+jobs:
+  update-xla-commit-hash:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          submodules: false
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
just curious--why do we need the token here?

Reply:
apparently we don't, i just copied this from somewhere else and assumed it was needed lol